advertising101.md

```diff
@@ -0,0 +1,167 @@
+* Using dynamic creative optimization, I can tailor the message to this consumer and select my branding marketing messages that are geared around diaper fit, electing to send a video that explains why fit is so important and why ours is so good, in order to influence the awareness and perceptions this customer has of our brand
+* I can also choose to follow up with display advertising on this consumer’s phone because the identity graph that my DSP has allows me to understand multiple of this consumer’s devices without knowing the name, address, phone number, etc of this consumer. I don’t need to know that type of PII—it’s not relevant.
+* But, sequential messaging across channels is relevant – depending on the product/service being offered, customers will engage with a brand multiple times prior to purchasing (e.g. for most of retail, it’s 7 times), and each of those engagements are important in helping that consumer move towards solving their problem aka converting
+* So, now I have the right person, the right time, the right message, the right channels and the right reason. But, how do I prove that it was effective? Since this was part of a branding campaign, in partnership with my DSP and a branding survey provider, I have a branding study set up on the campaign, which will tell me whether the campaign was effective in boosting awareness and changing perceptions of my brand
+  * Using an onboarding provider, my DSP onboards the list of exposed users to the brand survey provider and the onboarding provider then provides the matched list to the brand survey provider, who uses this information to send out the surveys (they will send out the surveys to a group of exposed and to a group of non-exposed users to compare the results and provide lift between the groups)
+  * When a campaign is multi-channel (which almost all of them are), the brand survey provider will accommodate for the different combinations of exposures across the various different channels and providers, which means that they will need all partners i.e. the DSP, the social providers, and the OLV providers to provide their exposed users using their respective identifiers and identity graphs
+
+
+*Consumer*: I am looking for diapers that fit, so to see ads that solve my specific problem is not intrusive (assuming the ad format isn’t annoying). If these ads follow me around for a relatively short period of time, I’m okay with that. If I had to instead pay for the publisher.com content and not see any ads, that would be okay as long as that was my choice. The same with my relationship with browser.com  goal is to show
```

Comment:
> The same with my relationship with browser.com  goal is to show

This seems to trail off. Not sure if there was more to this sentence?

Reply:
Ooops! corrected.